### PR TITLE
Update Makefile dev binaries

### PR DESCRIPTION
**Pull request by mresvanis**
This change updates the following tools:

- controller-gen
- kustomize
- operator-registry (OPM)

Signed-off-by: Michail Resvanis <mresvani@redhat.com>

### CODE REVIEW

**Comment by mresvanis** [comment]
/hold 
We should bump Go to 1.18 before merging this PR.

**Review by fabiendupont — APPROVED** [review]
/lgtm

**Pull request merged**
